examples/camera-controller/args.gni

```diff
@@ -34,3 +34,4 @@ chip_tlv_validate_char_string_on_read = true
 chip_tlv_validate_char_string_on_write = true
 
 chip_build_controller_dynamic_server = true
+chip_device_config_enable_wifipaf = false
```

Comment:
This shouldn't be needed, right?  Presumably it's because SIGTERM handling is being added which collides with the PAF handling?  See also https://github.com/project-chip/connectedhomeip/pull/38982

For now, maybe track removing this in an issue?

Reply:
It is needed, the signal handler implemented in PAF only work in server side example, and trigger immediately failure in controller app, such as camera-controller